src/router.ts

```diff
@@ -65,4 +67,12 @@ const router = createRouter({
   }
 });
 
+router.afterEach(to => {
+  mixpanel.track('Page View', {
```

Comment:
Shouldn't we use the default function `mixpanel.track_pageview` for page views? See: https://docs.mixpanel.com/docs/tracking-methods/sdks/javascript#tracking-page-views

Reply:
There seem to be a "built-in" event for page view, in my dashboard it show the data to 0, while if I track all event I can see some metric for "Page View" custom event

Reply:
> The track_pageview option currently does not auto-track page views in single-page applications. For single-page applications or other manual page view tracking, Mixpanel offers the standard page view event through mixpanel.track_pageview(). This call can be inserted into any hooks or listeners for your framework of choice.

It says auto-tracking doesn't work for SPAs.

I changed it to use `track_pageview`, I left `page_name` and `page_path` as they don't have default replacement (we have full URL, so maybe `page_path` is redundant a bit).

Reply:
Seem to work for me